hera_pspec/pspecdata.py

```diff
@@ -864,44 +905,40 @@ def R(self, key):
 
         # Only add to Rkey if a particular mode is enabled
         # If you do add to this, you need to specify this in self.set_R docstring!
-        Rkey = key + (self.data_weighting,) + (self.taper,)
+        Rkey = key + (self.taper,)
         if self.data_weighting == 'dayenu':
             # add extra dayenu params
             Rkey = Rkey + tuple(self.filter_extension,) + (self.spw_Nfreqs,) \
                    + (self.symmetric_taper,)
 
+        Rkey = Rkey + (self.data_weighting,)
         if Rkey not in self._R:
             # form sqrt(taper) matrix
             if self.taper == 'none':
-                sqrtT = np.ones(self.spw_Nfreqs).reshape(1, -1)
+                myT = np.ones(self.spw_Nfreqs)
             else:
-                sqrtT = np.sqrt(dspec.gen_window(self.taper, self.spw_Nfreqs)).reshape(1, -1)
-
+                myT = dspec.gen_window(self.taper, self.spw_Nfreqs, normalization='rms')
```

Comment:
why is this tapering normalization appropriate?

Reply:
This normalization sets the noise equivalent bandwidth equal to one. It isn't strictly necessary since the NEB is divided out at the normalization step but it makes \hat{q} a little more consistent. I can remove it if people want.

Reply:
I think we are used to thinking about applying tapers with peaks normalized to 1, and then propagating their normalization through to M. Perhaps good to keep it as is defined in master if it doesn't make a difference.

Reply:
Will do.